test/references/DataCite Ecology.txt

```diff
@@ -1,20 +1,12 @@
 register(DataDep(
-    "Data from Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation",
+    "Data from: Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation",
     """
-    	Dataset: Data from: Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation
-	Website: https://doi.org/10.5061/dryad.74699
-	Author: Eric J. B. Von Wettberg et al.
-	License: https://creativecommons.org/publicdomain/zero/1.0/
-	Date: 2018
-	
-	
-	
-	Please cite this dataset:
-	Von Wettberg, E. J. B., Chang, P. L., Başdemir, F., Carrasquila-Garcia, N., Korbu, L., Moenga, S. M., … Cook, D. R. (2018). Data from: Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation [Data set]. Dryad Digital Repository. https://doi.org/10.5061/dryad.74699
-	and this paper:
-	Von Wettberg, E. J. B., Chang, P. L., Başdemir, F., Carrasquila-Garcia, N., Korbu, L. B., Moenga, S. M., … Cook, D. R. (2018). Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation. Nature Communications, 9(1). doi:10.1038/s41467-018-02867-z
-	if you use this in your research.
-    """,
-    String["PUT DOWNLOAD URL HERE"],
-    
-))
+Website: https://doi.org/10.5061/dryad.74699
+Author: 	Eric J. B. Von Wettberg et al.
+License: 	https://creativecommons.org/publicdomain/zero/1.0/
+Date of Publication: 	2018
+Please cite this paper: 	Von Wettberg, E. J. B., Chang, P. L., Başdemir, F., Carrasquila-Garcia, N., Korbu, L. B., Moenga, S. M., … Cook, D. R. (2018). Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation. Nature Communications, 9(1). doi:10.1038/s41467-018-02867-z
```

Comment:
I think the label for `papercite` and `datacite`  should begin with an extra newline.
i.e. "\nPlease cite this paper:"